BE/teamgu/src/main/java/com/teamgu/database/repository/UserPoolRepositoryImpl.java

```diff
@@ -25,9 +27,16 @@
 
     @Override
     public List<Object[]> findUsersByFilter(UserPoolReqDto userPoolReqDto) {
-        studentNum = userPoolReqDto.getStudentNumber(); //무조건 들어와야하는 값
-        prjCode = userPoolReqDto.getProject(); //무조건 들어와야하는 값
-        stage = userPoolReqDto.getStudentNumber().substring(0, 2); //무조건 들어와야하는 값
+        log.info(userPoolReqDto);
+
+        try {
+            studentNum = userPoolReqDto.getStudentNumber(); //무조건 들어와야하는 값
+            prjCode = userPoolReqDto.getProject(); //무조건 들어와야하는 값
+            stage = userPoolReqDto.getStudentNumber().substring(0, 2); //무조건 들어와야하는 값
```

Comment:
```suggestion
            log.info(userPoolReqDto.getStudentNumber());
            stage = userPoolReqDto.getStudentNumber().substring(0, 2); //무조건 들어와야하는 값
```
추가해주면 값 확인하는데 좋을 것 같습니다~

Reply:
아 저부분 맨위에 log.info로 객체 toString 할수 있게 처리해놨습니다~!